tests/agent/test_agent.py

```diff
@@ -35,68 +41,79 @@
 
 
 @pytest.fixture
-def set_required_env_vars(monkeypatch, org_id, token):
+def set_required_env_vars(monkeypatch, org_id, token, localhost):
     monkeypatch.setenv("GX_CLOUD_ORGANIZATION_ID", org_id)
     monkeypatch.setenv("GX_CLOUD_ACCESS_TOKEN", token)
+    monkeypatch.setenv("GX_CLOUD_BASE_URL", localhost)
 
 
 @pytest.fixture
-def set_required_env_vars_missing_token(monkeypatch, org_id):
+def set_required_env_vars_missing_token(monkeypatch, org_id, localhost):
     monkeypatch.setenv("GX_CLOUD_ORGANIZATION_ID", org_id)
+    monkeypatch.setenv("GX_CLOUD_BASE_URL", localhost)
 
 
 @pytest.fixture
-def set_required_env_vars_missing_org_id(monkeypatch, token):
+def set_required_env_vars_missing_org_id(monkeypatch, token, localhost):
     monkeypatch.setenv("GX_CLOUD_ACCESS_TOKEN", token)
+    monkeypatch.setenv("GX_CLOUD_BASE_URL", localhost)
 
 
 @pytest.fixture
 def gx_agent_config(
-    set_required_env_vars, queue, connection_string, org_id, token
+    set_required_env_vars, queue, connection_string, org_id, token, localhost
 ) -> GXAgentConfig:
     config = GXAgentConfig(
         queue=queue,
         connection_string=connection_string,
         gx_cloud_access_token=token,
         gx_cloud_organization_id=org_id,
+        gx_cloud_base_url=localhost,
     )
     return config
 
 
 @pytest.fixture
 def gx_agent_config_missing_token(
-    set_required_env_vars_missing_token, queue, connection_string, org_id, token
+    set_required_env_vars_missing_token, queue, connection_string, org_id, token, localhost
 ) -> GXAgentConfig:
     config = GXAgentConfig(
         queue=queue,
         connection_string=connection_string,
-        gx_cloud_access_token=token,
         gx_cloud_organization_id=org_id,
+        token=token,
+        gx_cloud_base_url=localhost,
     )
     return config
 
 
 @pytest.fixture
 def gx_agent_config_missing_org_id(
-    set_required_env_vars_missing_org_id, queue, connection_string, org_id, token
+    set_required_env_vars_missing_org_id, queue, connection_string, org_id, token, localhost
 ) -> GXAgentConfig:
     config = GXAgentConfig(
         queue=queue,
         connection_string=connection_string,
         gx_cloud_access_token=token,
-        gx_cloud_organization_id=org_id,
+        org_id=org_id,
+        gx_cloud_base_url=localhost,
     )
     return config
 
 
+@pytest.fixture
+def localhost():
+    return "http://localhost:5000/"
+
+
 @pytest.fixture
 def org_id():
```

Comment:
```suggestion
def org_id_env_var():
```
maybe rename these fixtures so its easy to tell in test that it depends on host machines env